transcom/mymove
# B-18830 - Error message handling changes

## [I-12812](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A942271)

## Summary

This MR introduces error message handling changes. The UI now takes and error details provided from the server side and displays it.

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Office Application
2. Select Request Account
3. Fill out form but use an edipi that has already been used
4. Should see a more descriptive error message that comes from the server side

### Frontend


## Screenshots
![image](https://github.com/transcom/mymove/assets/136514590/9cc781b2-1451-43a9-b837-f72a945e6a33)

![image](https://github.com/transcom/mymove/assets/136514590/efc5827f-b0eb-45ed-bee2-ddc133b3cda9)
